.github/workflows/check-formatting-llpc.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   clang-format-check:
     name: clang-format
-    runs-on: "ubuntu-20.04"
+    runs-on: "ubuntu-22.04"
```

Comment:
I am not sure we can install clang-format-11 in 22.04.
And the opensource sanitization uses clang-format-11 in internal jenkins pipeline, because jenkins agents are 20.04.
Sometimes, this will cause clang-format check fail.

Reply:
Not sure what you mean, the job runs through on this PR, so clang-format-11 exists in 22.04? (See the package here: https://packages.ubuntu.com/search?keywords=clang-format-11&searchon=names&suite=jammy&section=all)